standalone_examples/dnf.py

```diff
@@ -30,13 +30,18 @@
 
 if host.fact.linux_name in ['CentOS', 'RedHat']:
 
-    # Note: Ignore the error if already installed
-    dnf.rpm(
-        {'Install EPEL rpm to enable EPEL repo'},
-        'https://dl.fedoraproject.org/pub/epel/epel-release-latest-'
-        '{{  host.fact.linux_distribution.major }}.noarch.rpm',
-        ignore_errors=True,
-    )
+    packages = host.fact.rpm_packages
+    epel_installed = False
+    for p in packages.keys():
+        if p.startswith('epel-release'):
+            epel_installed = True
+
+    if not epel_installed:
```

Comment:
I think the real fix here is to fix the behaviour of `dnf.rpm` and `yum.rpm` to properly check, within the operation, if the RPM is installed and handle appropriately (whether upgrade or downgrade). I'll investigate this as part of #208.

Reply:
Ok. I'll leave it like this for now.